test/index.js

```diff
@@ -173,36 +176,25 @@ function getLog(name, stream, level) {
     }));
 }
 
-function dtrace() {
-    var dtp;
-
-    try {
-        var d = require('dtrace-provider');
-        dtp = d.createDTraceProvider('restifyUnitTest');
-    } catch (e) {
-        dtp = null;
-    }
-    return (dtp);
-}
-
 // --- Tests
 
 describe('restify-client tests', function () {
 
     before(function (callback) {
         try {
             SERVER = restify.createServer({
-                dtrace: dtrace,
```

Comment:
7.x no longer takes dtrace as a function, it's now a boolean. Removed as it wasn't super important for the client tests.